defaults/test_default.yaml

```diff
@@ -274,3 +274,35 @@ skip_test_stages: {}
 n_db_zero_token_nodes: 0
 zero_token_instance_type_db: 'i4i.large'
 use_zero_nodes: false
+
+latency_decorator_error_thresholds:
+  write:
+    default:
+      P90 write:
+        fixed_limit: 5
+      P99 write:
+        fixed_limit: 10
+      Throughput write:
```

Comment:
Why intruding this 2K default?
Do we have any data to back it up ?

Reply:
by default we measure latency with throttling - in that case dropping more than 2k I consider as a bug. And generally it never drops even by 500.